productImport/ontologyValidator/productsValidator.go

```diff
@@ -90,6 +90,14 @@ func (v *Validator) validateProductsAgainstRules(
 							isError = true
 						}
 
+						// units of measurement (UOM)
+						if attrUoM, ok := prodToMapped[v.buildUomColumn(attr.Name)]; ok {
+							if isValid, errorMessage := v.isValidAttributeUoM(attrUoM, attr); !isValid && errorMessage != "" {
```

Comment:
What happens if attribute is not valid and error message is empty?

Reply:
fixed